hermes/views.py

```diff
@@ -181,9 +180,137 @@ def get(self, request, *args, **kwargs):
+                        meandist: <>,
+                        meananom: <>,
+                        perihdist: <>,
+                        epochofperih: <>
+                    },
+                    discovery_info: {
+                        reporting_group: <>,
+                        discovery_source: <>,
+                        transient_type: <Type of source, one of PSN, nuc, PNV, AGN, or Other>,
+                        proprietary_period: <Duration that this discovery should be kept private>,
+                        proprietary_period_units: <Units for proprietary period, Days, seconds, Years>
+                    },
+                    redshift: <>,
+                    host_name: <Host galaxy name>,
+                    host_redshift: <>,
+                    aliases: [
+                        'alias1',
+                        'alias2',
+                        ...
+                    ],
+                    group_associations: <String of group associations for this target>
+                }
+            ],
+            photometry: [
+                {
+                    target_index: <Index of target list that this photometry relates to. Can be left out if there is only one target>,
+                    date_obs: <Date of the observation, in a parseable format or JD>,
+                    telescope: <Discovery telescope>,
+                    instrument: <Discovery instrument>,
+                    bandpass: <Wavelength band of the discovery observation>,
+                    brightness: <Brightness of the discovery>,
+                    brightness_error: <Brightness error of the discovery>,
+                    brightness_unit: <Brightness units for the discovery,
+                                      current supported values: [AB mag, Vega mag, mJy, erg / s / cm² / Å]>,
+                    new_discovery: <Boolean if this photometry is for a new discovery or not>
+                    exposure_time: <Exposure time in seconds for this photometry>,
+                    observer: <The entity that observed this photometry data>,
+                    comments: <String of comments for the photometry>,
+                    limiting_brightness: <The minimum brightness at which the target is visible>,
+                    limiting_brightness_unit: <Unit for the limiting brightness>
+                    group_associations: <>
+                }
+            ],
+            spectroscopy: [
+                {
+                    target_index: <Index of target list that this specotroscopic data relates to. Can be left out if there is only one target>,
+                    date_obs: <Date of the observation, in a parseable format or JD>,
+                    telescope: <specotroscopic data telescope>,
+                    instrument: <specotroscopic data instrument>,
+                    setup: <>,
+                    flux: {
+                        value: <Flux value of the specotroscopic data>,
```

Comment:
If I understand the above correctly, we should probably replace "data" with "datum" for all of these entries.